ci/verify_bundle.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import subprocess
+
+import torch
+from utils import download_large_files, get_changed_bundle_list
+
+
+def verify_download_large_files(bundle_path: str):
+    """
+    This function is used to verify the `large_files` file (if exists) of the input bundle path.
+    Wrong download link or checksum will raise an error.
+    A successful verification will also keep the downloaded large files for other verifications.
+
+    """
+    # verify download
+    try:
+        for large_file_type in [".yml", ".yaml", ".json"]:
+            large_file_name = "large_files" + large_file_type
+            large_file_path = os.path.join(bundle_path, large_file_name)
+            if os.path.exists(large_file_path):
+                download_large_files(bundle_path=bundle_path, large_file_name=large_file_name)
+    except Exception as e:
+        raise ValueError(f"Download large files in {bundle_path} error") from e
+
+
+def verify_metadata_format(bundle_path: str):
+    """
+    This function is used to verify the metadata format.
+
+    """
+    verify_cmd = "python -m monai.bundle verify_metadata --meta_file configs/metadata.json --filepath eval/schema.json"
+
+    subprocess.check_call(f"{verify_cmd}", shell=True, cwd=bundle_path)
```

Comment:
If there are some testing errors, subprocess.check_call might not print detailed error messages, this could increase the difficulty of debugging. So perhaps we change to use native python commands for this?